frontend/src/router.js

```diff
@@ -781,12 +828,12 @@ export function categoryRoute(camp, category, query = {}) {
 }
 
 export function checklistRoute(camp, checklist, query = {}) {
-  if (camp._meta.loading || checklist._meta.loading) return {}
+  if (camp?._meta.loading || checklist._meta.loading) return {}
   return {
-    name: 'admin/checklists/checklist',
+    name: camp ? 'camp/admin/checklists/checklist' : 'admin/checklists/checklist',
     params: {
-      campId: camp.id,
-      campTitle: slugify(camp.title),
+      campId: camp?.id,
+      campTitle: slugify(camp?.title ?? ''),
       checklistId: checklist.id,
       checklistName: slugify(checklist.name),
     },
```

Comment:
Please make an early return if the camp is null and you want to go to the admin interface